sdk/cosmos/azure-cosmos/tests/test_backwards_compatibility.py

```diff
@@ -78,6 +81,105 @@ def test_populate_partition_key_range_statistics(self):
         except StopIteration:
             pass
 
+    def test_session_token_compatibility(self):
+        # Verifying that behavior is unaffected across the board for using `session_token` on irrelevant methods
+        # Database
+        database = self.client.create_database(str(uuid.uuid4()), session_token=str(uuid.uuid4()))
+        assert database is not None
+        database2 = self.client.create_database_if_not_exists(str(uuid.uuid4()), session_token=str(uuid.uuid4()))
+        assert database2 is not None
+        database_list = list(self.client.list_databases(session_token=str(uuid.uuid4())))
+        database_list2 = list(self.client.query_databases(query="select * from c", session_token=str(uuid.uuid4())))
+        assert len(database_list) > 0
+        assert database_list == database_list2
+        database_read = database.read(session_token=str(uuid.uuid4()))
+        assert database_read is not None
+        self.client.delete_database(database2.id, session_token=str(uuid.uuid4()))
+        try:
+            database2.read()
+            pytest.fail("Database read should have failed")
+        except CosmosHttpResponseError as e:
```

Comment:
Doesn't need changing - but just in case, thought I'd mention that another way to represent this logic:
```python
with pytest.raises CosmosHttpResponseError as e:
    database2.read()
assert e.value.status_code == 404
```